po/pt.po

```diff
@@ -167,89 +206,150 @@ msgstr ""
 "Introduza uma palavra-chave para procurar em 'extensions.gnome.org' "
 "extensões do GNOME Shell."
 
-#: src/exm-browse-page.blp:22
-msgid "e.g. \"Blur my Shell\""
-msgstr "ex: \"Blur my Shell\""
-
 #. Translators: dropdown items for sorting search results
-#: src/exm-browse-page.blp:33
+#: src/exm-browse-page.blp:31
 msgid "Popularity"
 msgstr "Popularidade"
 
-#: src/exm-browse-page.blp:33
+#: src/exm-browse-page.blp:31
 msgid "Downloads"
 msgstr "Transferências"
 
-#: src/exm-browse-page.blp:33
+#: src/exm-browse-page.blp:31
 msgid "Recent"
 msgstr "Recentes"
 
-#: src/exm-browse-page.blp:33
+#: src/exm-browse-page.blp:31
 msgid "Name"
 msgstr "Nome"
 
-#: src/exm-browse-page.blp:60
+#: src/exm-browse-page.blp:58
+msgid "Load More Results"
+msgstr "Carregar mais resultados"
+
+#: src/exm-browse-page.blp:68
 msgid "No Results Found"
 msgstr "Nenhum resultado"
 
-#: src/exm-comment-dialog.blp:35 src/exm-detail-view.blp:35
-#: src/exm-detail-view.blp:168 src/exm-detail-view.c:350
+#. Translators:
+#. - '%s' is an extension e.g. Blur my Shell
+#. - Please use unicode quotation marks e.g. “” (not "")
+#: src/exm-browse-page.c:292
+#, c-format
+msgid "e.g. “%s”"
+msgstr "por exemplo, \"%s\""
```

Comment:
```suggestion
msgstr "por exemplo, “%s”"
```

Could we use unicode quotation marks (“”) here?

Reply:
Fixed